app/controllers/openid_connect/certs_controller.rb

```diff
@@ -0,0 +1,9 @@
+module OpenidConnect
+  class CertsController < ApplicationController
+    def index
+      expires_in 1.week, public: true
```

Comment:
will this make adding new SPs problematic as far as cache invalidation is concerned?